package.json

```diff
@@ -133,6 +125,9 @@
     "tldts-experimental": "5.7.75",
     "webextension-polyfill": "0.9.0"
   },
+  "overrides": {
+    "ansi-html": "0.0.8"
```

Comment:
Is it safe to move back to ansi-html from the community version? Maybe we can stick to the current version?

Reply:
Unfortunately broccoli hasn't been updated yet and so it pulls in an old version of ansi-html without this override.

Reply:
What I mean is that this PR is changing the package that is used for the override. We used to override with https://www.npmjs.com/package/ansi-html-community, but after this PR we now override with https://www.npmjs.com/package/ansi-html. This is the original one but it was unmaintained for a long time, which led to the community fork as far as I remember. Unless there is an issue with the community fork I'd rather keep the same package in this PR. Wdyt?

Reply:
I see what you mean. Yes, I had not noticed that we had switched to a different npm package. It does appear that the `-community` fork is now less up-to-date than the original [`ansi-html` package](https://www.npmjs.com/package/ansi-html) which is now at version 0.0.9.

So I think we should probably switch back to the original package. But you are correct, this is a change from what we were doing.

Reply:
I checked the changes in the original package and it seems fine. Looks like the latest version also drops lodash as dev dependency to fix a vulnerability.